src/overlay/test/OverlayTests.cpp

```diff
@@ -2141,33 +2141,32 @@ TEST_CASE("overlay flow control", "[overlay][flowcontrol]")
         simulation->addPendingConnection(vNode2NodeID, vNode3NodeID);
         simulation->addPendingConnection(vNode3NodeID, vNode1NodeID);
 
-        for (auto& node2 : simulation->getNodes())
+        simulation->startAllNodes();
+        simulation->crankUntil(
+            [&] { return simulation->haveAllExternalized(2, 1); },
+            3 * Herder::EXP_LEDGER_TIMESPAN_SECONDS, false);
+        if (appProtocolVersionStartsFrom(*simulation->getNodes()[0],
+                                         SOROBAN_PROTOCOL_VERSION))
         {
-            if (appProtocolVersionStartsFrom(*node2, SOROBAN_PROTOCOL_VERSION))
-            {
-                modifySorobanNetworkConfig(
-                    *node2, [](SorobanNetworkConfig& cfg) {
-                        cfg.mTxMaxSizeBytes =
-                            MinimumSorobanNetworkConfig::TX_MAX_SIZE_BYTES;
-                    });
-            }
+            upgradeSorobanNetworkConfig(
+                [](SorobanNetworkConfig& cfg) {
+                    cfg.mTxMaxSizeBytes =
+                        MinimumSorobanNetworkConfig::TX_MAX_SIZE_BYTES;
+                },
+                simulation);
         }
-        simulation->startAllNodes();
     };
 
     SECTION("enabled")
     {
         setupSimulation();
 
-        simulation->crankUntil(
-            [&] { return simulation->haveAllExternalized(2, 1); },
-            3 * Herder::EXP_LEDGER_TIMESPAN_SECONDS, false);
         // Generate a bit of load to flood transactions, make sure nodes can
         // close ledgers properly
         auto& loadGen = node->getLoadGenerator();
-        loadGen.generateLoad(
-            GeneratedLoadConfig::createAccountsLoad(/* nAccounts */ 150,
-                                                    /* txRate */ 1));
+        loadGen.generateLoad(GeneratedLoadConfig::createAccountsLoad(
+            /* nAccount */ 150,
```

Comment:
please revert this

Reply:
the typo is still there, should be nAccounts

Reply:
oops I thought you meant the formatting 😅 done.